store/tikv/txn.go

```diff
@@ -393,7 +405,9 @@ func (txn *KVTxn) onCommitted(err error) {
 		if err != nil {
 			info.ErrMsg = err.Error()
 		}
-		txn.commitCallback(info, err)
+		infoStr, err2 := json.Marshal(info)
+		_ = err2
```

Comment:
Does it seem we need to keep the err2? the related test case is `TestTiDBLastTxnInfo`

Reply:
The test is not related to this. err2 should always be nil.